src/errors/checkXcmTxInputs.ts

```diff
@@ -57,6 +57,7 @@ export const checkAssetIdInput = (
 			const isNativeChain =
 				chainInfo.specName.toLowerCase() === specName.toLowerCase();
 
+			// TODO: This logic breaks when we have 2 chains that share the same native tokens.
```

Comment:
@marshacb So this logic below has an edge case we didn't cover. Such as what if the 2 chains are statemine, and polkadot. It will fail since they both share native tokens. 

You can see the failing tests in the test suite for github actions.

Reply:
Hey so I removed it, as I think the check might be a little redundant. Let me know what you think. We can add it back in when we do specific checks on direction.